crates/bootstrap_srv/src/parse.rs

```diff
@@ -5,7 +5,7 @@ pub struct ParsedEntry {
     pub agent: ed25519_dalek::VerifyingKey,
 
     /// space
-    pub space: bytes::Bytes,
+    pub space_id: bytes::Bytes,
```

Comment:
This `ParsedEntry` type seems to be stored and parsed from temp files. Might this be a problem temporarily when bumping the server version on a deployed instance? If we think it's problematic I can revert it.